sherpa/astro/ui/utils.py

```diff
@@ -3789,8 +3789,7 @@ def get_backscal(self, id=None, bkg_id=None):
         References
         ----------
 
-        .. [1] "The OGIP Spectral File Format", Arnaud, K. & George, I.
-               http://heasarc.gsfc.nasa.gov/docs/heasarc/ofwg/docs/spectra/ogip_92_007/ogip_92_007.html
+        `Arnaud., K. & George, I., "The OGIP Spectral File Format" <http://heasarc.gsfc.nasa.gov/docs/heasarc/ofwg/docs/spectra/ogip_92_007/ogip_92_007.html>`_
```

Comment:
```suggestion
        `Arnaud., K. & George, I., "The OGIP Spectral File Format" <https://heasarc.gsfc.nasa.gov/docs/heasarc/ofwg/docs/spectra/ogip_92_007/ogip_92_007.html>`_
```

Reply:
I'm updating name and link in a separate commit.